src/Sylius/Component/Grid/Definition/Field.php

```diff
@@ -152,7 +152,11 @@ public function setEnabled($enabled)
      */
     public function setSortable($sortable)
     {
-        $this->sortable = $sortable ?: $this->name;
+        if ($sortable === true) {
+            $this->sortable = $this->name;
+        }
+
+        $this->sortable = $sortable;
```

Comment:
But `true` is evaluated to TRUE, but `$this->name` should be used so it is more like `$this->sortable = ($sortable === true) ? $this->name :  $sortable`